tests/src/ownership/tests/functions/owner.rs

```diff
@@ -15,7 +15,7 @@ mod success {
 
         assert!(matches!(
             owner(&owner1.contract).await,
-            State::Uninitialized()
+            State::Uninitialized
```

Comment:
Does this work? Last time I used this the SDK team said that it gets treated as a function call so we must `call` it.

Reply:
`State::Uninitialized()` results in an error